lib/update/phase_kubernetes.go

```diff
@@ -392,7 +392,7 @@ func supportsTaints(gravityPackage loc.Locator) (supports bool, err error) {
 
 func waitForEndpoints(ctx context.Context, client corev1.CoreV1Interface, server storage.Server) error {
 	clusterLabels := labels.Set{"app": defaults.GravityClusterLabel}
-	kubednsLegacyLabels := labels.Set{"k8s-app": "kube-dns"}
+	kubednsLegacyLabels := labels.Set{"k8s-app": "kubedns"}
```

Comment:
Now both legacy and [current](https://github.com/gravitational/gravity/pull/113/files#diff-f9e91517c31865c93494fba9442eeff9R807) DNS labels are `kubedns`

Reply:
You're watching too quickly, I did update both, just missed committing the update to defaults in the same push.